cyber/mainboard/module_argument.cc

```diff
@@ -62,7 +62,6 @@ void ModuleArgument::ParseArgument(const int argc, char* const argv[]) {
 }
 
 void ModuleArgument::GetOptions(const int argc, char* const argv[]) {
-  opterr = 0;  // extern int opterr
```

Comment:
Add Cyber folks to check if it's safe to remove this.

Reply:
It seems like intended behavior: https://linux.die.net/man/3/opterr

Reply:
Sorry for my too late response. IMO, the purpose of opterr is intended to prevent error message for getopt(3) rather than getopt_long(3).

Reply:
opterr is used for both getopt(3) and getopt_long(3).
The  getopt_long()  function works like getopt() except that it also accepts long options.

Reply:
Umm...I  missed that and you are right @jinghaomiao @GoLancer @xiaoxq, will close this PR.